cmd/all.go

```diff
@@ -135,3 +111,29 @@ func splitFullName(fullName string) (string, string) {
 	s := strings.Split(fullName, "/")
 	return s[0], s[1]
 }
+
+func initPrometheus() prometheus.Counter {
```

Comment:
maybe we could move the Prometheus funcs in a prometheus package

Reply:
I'll add a "metrics" package. It uses PrometheusCounter() as prometheus.Counter generator.